unixfs/mod/dagmodifier_test.go

```diff
@@ -750,3 +750,11 @@ func BenchmarkDagmodWrite(b *testing.B) {
 		}
 	}
 }
+
+func getOffset(reader uio.DagReader) int64 {
```

Comment:
I'm rewriting `getOffset` from `dagreader_test.go`, exporting it doesn't seem to work, is there a cleaner way (without adding it back to `dagreader.go`)?